<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generate_powClass</class>
 <widget class="QMainWindow" name="Generate_powClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>333</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate_pow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #36454f;
}

QLabel {
	color: #ffffff;
}

#title {
	font: 30pt bold;
}

#label {
	border: 1px solid #ffffff;
}

QSpinBox {
	background-color: #36454f;
	color: #ffffff;
	selection-background-color: #65816d;
}

QSpinBox::up-button {
	border-top: 1px solid #ffffff;
	border-left: 1px solid #ffffff;
	border-right: 1px solid #ffffff;
	border-bottom: 0.5px solid #ffffff;
	background-color: #36454f;
}

QSpinBox::up-arrow {
	image: url(C:/Users/marem/Downloads/vectorpaint (9).svg);
}

QSpinBox::down-button {
	border-top: 0.5px solid #ffffff;
	border-left: 1px solid #ffffff;
	border-right: 1px solid #ffffff;
	border-bottom: 1px solid #ffffff;
	background-color: #36454f;
}

QSpinBox::down-arrow {
	image: url(C:/Users/marem/Downloads/vectorpaint (10).svg);
}

QSlider::groove:vertical {
	border: 1px solid #ffffff;
    /*border: 1px solid #ffa500;*/
    /*height: 8px;*/
    background: #36454f;/*#ff8243;*/
    margin: 2px 0;
}

QSlider::handle:vertical {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ffffff, stop:1 #a0a0a0);
	/*background: #800020;*/
	/*border: 1px solid #5c5c5c;*/
	height: 8px;
    margin: -2px 0;
    /*border-radius: 3px;*/
}

QPushButton {
	border: 1px solid #ffffff;
	background-color: #36454f;
	color: #ffffff;
	border-radius: 5px;
}

QPushButton::hover {
	border: 1px solid #ffffff;
	background-color: #65816d;
	color: #ffffff;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>63</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Doubler</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Gen_pow.qrc"/>
 </resources>
 <connections/>
</ui>
